### DISCUSSION TITLE
Supabase Dashboard locally?

### DISCUSSION THREAD

**Discussion by ctwhome** [supabase/supabase#772] [Questions]
Hi there, I can't find the way to open the supabase dashboard locally with the new supabase cli. I guess that this feature is not available? 

`supabase init`
`upabase dev`

Thanks!

**Comment by kiwicopple** [comment]
It's not available on the CLI unfortunately @ctwhome - in fact the CLI is still running a bit behind schedule. I'll put a bit more work into it in March

**Reply by ctwhome** [reply]
awesome, thank you!
